dtmi/stmicroelectronics/steval_proteus1/standard_if-1.json

```diff
@@ -0,0 +1,301 @@
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Gyroscope X/Y/Z components [dps]",
+      "displayName": {
+        "en": "Gyro Value [dps]"
+      },
+      "name": "gyroscope",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Single angular velocity sample. I.e.: {g_x, g_y, g_z}."
+        },
+        "displayName": {
+          "en": "Single angular velocity sample."
+        },
+        "fields": [
+          {
+            "name": "g_x",
+            "schema": "double"
+          },
+          {
+            "name": "g_y",
+            "schema": "double"
+          },
+          {
+            "name": "g_z",
+            "schema": "double"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Acceleration Peak Status X/Y/Z components [m/sˆ2]",
+      "displayName": {
+        "en": "Acc Peak Status [m/sˆ2]"
+      },
+      "name": "predictiveaccelerationstatus",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Sample format I.e.: {statusacc_x, statusacc_y, statusacc_z, accpeak_x, accpeak_y, accpeak_z}."
+        },
+        "displayName": {
+          "en": "Single Predictive Acceleration Peak Status sample."
+        },
+        "fields": [
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration X:"
+            },
+            "name": "statusacc_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration Y:"
+            },
+            "name": "statusacc_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[AccPeak] Status Acceleration Z:"
+            },
+            "name": "statusacc_z",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Acceleration Peak X:"
+            },
+            "name": "accpeak_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Acceleration Peak Y:"
+            },
+            "name": "accpeak_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Acceleration Peak Z:"
+            },
+            "name": "accpeak_z",
+            "schema": "double"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Speed Status X/Y/Z components [mm/s]",
+      "displayName": {
+        "en": "Speed Status  [m/sˆ2]"
+      },
+      "name": "predictivespeedstatus",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Sample format I.e.: {statusacc_x, statusacc_y, statusacc_z, rmsspeed_x, rmsspeed_y, rmsspeed_z}."
+        },
+        "displayName": {
+          "en": "Single Predictive Speed Status sample."
```

Comment:
capitalize "sample" and remove terminal period

Reply:
done